### PR TITLE
Implement browser provider

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @inikulin @VasilyStrelyaev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+npm install -g testcafe-browser-provider-phantomjs
+```
+
+## Usage
+
+When you run tests from the command line, use the provider name when specifying the browsers:
+
+```
+testcafe chrome,phantomjs 'path/to/test/file.js'
+```
```
Comment (inikulin) [inline README.md:18]: Same notes as for provider here

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# testcafe-browser-provider-phantomjs
+[![Build Status](https://travis-ci.org/DevExpress/testcafe-browser-provider-phantomjs.svg)](https://travis-ci.org/DevExpress/testcafe-browser-provider-phantomjs)
+
+This is the **phantomjs** browser provider plugin for [TestCafe](http://devexpress.github.io/testcafe).
```
Comment (inikulin) [inline README.md]: We need a link to phantomjs website

**Comment by inikulin** [comment]
\r-

**Comment by inikulin** [comment]
Also, don't forget to add GitHub repo descriptions as it's done for providers

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+var assert = require('assert');
+
+
+describe('Test suite', function () {
```
Comment (inikulin) [inline test/test.js]: Where are the tests?

File: README.md (hunk 2 above)
Comment (VasilyStrelyaev) [inline README.md]: so it's okay to alter a generated readme?

Comment (inikulin) [inline README.md]: Of course, it's just a boilerplate

**Comment by VasilyStrelyaev** [comment]
all texts are in the generator, so seems to be nothing to review here for me

**Comment by AndreyBelym** [comment]
@VasilyStrelyaev, I will appreciate your help with repository description.

**Comment by VasilyStrelyaev** [comment]
I'd use descriptions similar to those of the reporters. I'll do it.

**Comment by AndreyBelym** [comment]
Despite the module `phantom` is `babel`ified ES6, it doesn't polyfill the `Map` class, which prevents it to run on `node@0.10`. Do we need to create a fork, or just drop `node@0.10` too?

**Comment by inikulin** [comment]
`phantom` works from `v0.12` onward. We can declare support starting from this version too: add `engine` field in `package.json` and highlight it in readme

**Comment by AndreyBelym** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

File: utils/spawn.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+var nodeSpawn = require('child_process').spawn;
+var Promise   = require('pinkie');
+var assign    = require('lodash').assign;
+
+
+module.exports = function (cmd, args, envs) {
+    return new Promise(function (resolve, reject) {
```
Comment (inikulin) [inline utils/spawn.js:7]: Why not use on of many existing utility modules for this? E.g. https://github.com/inikulin/cp-sugar ?

Comment (inikulin) [inline utils/spawn.js:7]: Oh, I see, you need to pass `env` around

**Comment by inikulin** [comment]
lgtm

**Pull request merged**